drivers/wifi/nxp/nxp_wifi_drv.c

```diff
@@ -1343,18 +1343,23 @@ static int nxp_wifi_power_save(const struct device *dev, struct wifi_ps_params *
 			}
 			break;
 		case WIFI_PS_PARAM_MODE:
+			
 			if (params->mode == WIFI_PS_MODE_WMM) {
+				#if CONFIG_NXP_WIFI_WMM_UAPSD
```

Comment:
```suggestion
#ifdef CONFIG_NXP_WIFI_WMM_UAPSD
```

Reply:
same for other `#if CONFIG_NXP_WIFI_WMM_UAPSD`

Reply:
This modification was added to prevent compilation errors when the “Disable WMM feature” configuration is applied.